python/core/layertree/qgslayertreenode.sip

```diff
@@ -1,194 +1,314 @@
+%Docstring
+Emitted when one or more nodes will be added to a node within the tree
+%End
     void addedChildren( QgsLayerTreeNode *node, int indexFrom, int indexTo );
-    //! Emitted when one or more nodes will be removed from a node within the tree
+%Docstring
+Emitted when one or more nodes have been added to a node within the tree
+%End
     void willRemoveChildren( QgsLayerTreeNode *node, int indexFrom, int indexTo );
-    //! Emitted when one or more nodes has been removed from a node within the tree
+%Docstring
+Emitted when one or more nodes will be removed from a node within the tree
+%End
     void removedChildren( QgsLayerTreeNode *node, int indexFrom, int indexTo );
-    //! Emitted when check state of a node within the tree has been changed
+%Docstring
+Emitted when one or more nodes has been removed from a node within the tree
+%End
     void visibilityChanged( QgsLayerTreeNode *node );
-    //! Emitted when a custom property of a node within the tree has been changed or removed
-    void customPropertyChanged( QgsLayerTreeNode *node, const QString& key );
-    //! Emitted when the collapsed/expanded state of a node within the tree has been changed
+%Docstring
+Emitted when check state of a node within the tree has been changed
+%End
+    void customPropertyChanged( QgsLayerTreeNode *node, const QString &key );
+%Docstring
+Emitted when a custom property of a node within the tree has been changed or removed
+%End
     void expandedChanged( QgsLayerTreeNode *node, bool expanded );
-    //! Emitted when the name of the node is changed
-    //! @note added in 3.0
-    void nameChanged( QgsLayerTreeNode* node, QString name );
+%Docstring
+Emitted when the collapsed/expanded state of a node within the tree has been changed
+%End
+    void nameChanged( QgsLayerTreeNode *node, QString name );
+%Docstring
+.. versionadded:: 3.0
+%End
 
   protected:
 
-    QgsLayerTreeNode( NodeType t );
+    QgsLayerTreeNode( NodeType t, bool checked = true );
+%Docstring
+Constructor
+%End
+    QgsLayerTreeNode( const QgsLayerTreeNode &other );
+
 
     void readCommonXml( QDomElement &element );
+%Docstring
+Read common XML elements.
+%End
     void writeCommonXml( QDomElement &element );
+%Docstring
+Write common XML elements.
+%End
 
-    //! Low-level insertion of children to the node. The children must not have any parent yet!
-    void insertChildrenPrivate( int index, QList<QgsLayerTreeNode*> nodes );
-    //! Low-level removal of children from the node.
+    void insertChildrenPrivate( int index, QList<QgsLayerTreeNode *> nodes );
+%Docstring
+Low-level insertion of children to the node. The children must not have any parent yet!
+%End
     void removeChildrenPrivate( int from, int count, bool destroy = true );
+%Docstring
+Low-level removal of children from the node.
+%End
 
-  private:
-    QgsLayerTreeNode( const QgsLayerTreeNode& other );
+  protected:
 
+  private:
+    QgsLayerTreeNode( const QgsLayerTreeNode &other );
```

Comment:
Not sure if I understand this change - earlier in the header file the copy constructor of QgsLayerTreeNode is declared as protected ... why would we want to declare it in SIP as private?